dotcom-rendering/src/web/components/App.tsx

```diff
@@ -242,7 +242,7 @@ export const App = ({ CAPI, NAV, ophanRecord }: Props) => {
 			const hasOptedOut = await hasOptedOutOfArticleCount();
 			if (!hasOptedOut) {
 				incrementDailyArticleCount();
-				incrementWeeklyArticleCount(storage.local, CAPI.pageId);
+				incrementWeeklyArticleCount(storage.local, CAPI.pageId, CAPI.config.keywordIds.split(','));
```

Comment:
I guess we should add `CAPI.config.keywordIds` to the dependencies array?

Reply:
done! In theory nothing on `CAPI` should ever change, but it's good to be thorough!